src/components/Badge/badge.css

```diff
@@ -39,13 +41,19 @@
   border: var(--sgds-border-width-1) solid var(--sgds-neutral-border-color-default);
 }
 
+sgds-tooltip {
+  display: block;
+  max-width: 100%;
```

Comment:
did you and @anzteng consider giving a absolute value for max width rather than 100%? 
 how come settle for percentage in the end